apps/Makefile.nrf_pca10001.mk

```diff
@@ -9,19 +9,24 @@ JLINK_EXE ?= JLinkExe
 
 # Apps to link may grow over time so defer expanding that
 .SECONDEXPANSION:
-$(TOCK_APP_BUILD_DIR)/kernel_and_app.elf: $(TOCK_BUILD_DIR)/ctx_switch.o $(TOCK_BUILD_DIR)/kernel.o $$(APPS_TO_LINK_TO_KERNEL) | $(TOCK_BUILD_DIR)
+$(TOCK_APP_BUILD_DIR)/kernel_and_app.elf: $(TOCK_BUILD_DIR)/ctx_switch.o $(TOCK_BUILD_DIR)/kernel.o $$(TOCK_BUILD_DIR)/kernel.o $$(APPS_TO_LINK_TO_KERNEL) | $(TOCK_BUILD_DIR)
```

Comment:
I don't recall changing this line, so I will have to read the file thoroughly to figure out whether it is needed- I'll do that now